spec/calendar.html

```diff
@@ -0,0 +1,151 @@
+<emu-clause id="sec-temporal-calendar-objects">
+  <h1>Temporal.Calendar Objects</h1>
+  <p>A Temporal.Calendar object is an immutable Object representing a calendar.</p>
+
+  <emu-clause id="sec-temporal-calendar-abstract-ops">
+    <h1>Abstract Operations for Temporal.Calendar Objects</h1>
+
+    <emu-clause id="sec-temporal-getbuiltincalendar" aoid="GetBuiltinCalendar">
```

Comment:
I wonder if the builtin calendars should be in ECMA-402 instead. The only calendar that an implementation is required to have, if not supporting Intl, is the ISO calendar. That would mean we don't have to have a `%Temporal.Calendar%.[[BuiltinCalendars]]`, we only need an intrinsic `%ISO8601Calendar%` object. We could have a prose description of what Temporal.Calendar.from does if the implementation doesn't have ECMA-402 similar to https://tc39.es/ecma262/#sec-number.prototype.tolocalestring

Reply:
I thought we could set/describe the exact values stored in the `%Temporal.Calendar%.[[BuiltinCalendars]]` in 402. This is because there isn't any precedent for overloading an abstract operation in 262, only for public methods. Perhaps we could add a static `Temporal.Calendar.getBuiltinCalendars` method that can be overloaded in 402 and `getDefaultCalendar` could call that method instead.

Reply:
Or perhaps I could make `Temporal.Calendar.from` overload-able in 402 and make `getDefaultCalendar` call `from` instead.

Reply:
I don't see any particular problem with overloading an abstract operation in 262, as long as we're clear in the spec text about what we're doing.

Reply:
@littledan alright then.

Reply:
@ptomato @littledan done, PTAL.